src/client/cmd/common_cli.cpp

```diff
@@ -102,3 +102,23 @@ mp::ReturnCode cmd::standard_failure_handler_for(const std::string& command, std
 
     return return_code_for(status.error_code());
 }
+
+bool cmd::update_available(const multipass::UpdateInfo& update_info)
+{
+    return update_info.version() != "";
+}
+
+std::string cmd::update_notice(const multipass::UpdateInfo& update_info)
+{
+    if (update_available(update_info))
```

Comment:
Should the consumer do this check instead maybe?

Reply:
Totally could. I tried this approach to keep logic out of the command implementations (there's already lots in there) but yeah, it's not great.

Reply:
Done.